### DISCUSSION TITLE
Custom page textarea vs textinput bug

### DISCUSSION THREAD

**Discussion by griggsk** [filamentphp/filament#13488] [Help]
### Package

Form builder

### Package Version

v3.2*

### How can we help you?

Upon update to filament latest, the textarea in my custom page stopped passing the entered value. 

- The same form in a filament resource works. 
- The same form with a textinput field replacing the textarea field in a custom page works. 
- But the textarea in the custom page always passes null to my create method.

I'm not sure what to do. Ideas?

WORKS
```
TextInput::make('prompt_'.$item->id)
                        ->label(trans($prompt->label))
                        ->columnStart(['sm' => 1, 'lg' => 2])
                        ->columnSpan(['sm' => 1, 'lg' => 9]);
```

DOES NOT WORK
```
Textarea::make('prompt_'.$item->id)
                        ->label(trans($prompt->label))
                        ->columnStart(['sm' => 1, 'lg' => 2])
                        ->columnSpan(['sm' => 1, 'lg' => 9]);
```

In model
```
protected $fillable = [
        'prompt_id',
        'value',
    ];
```

**Comment by leandrocfe** [comment]
Could you share the code?

**Reply by kroberts-highwater** [reply]
I have the exact same issue. It comes with a console error too. "Uncaught ReferenceError: state is not defined" on livewire.js

**Comment by griggsk** [comment]
Sure. Here is the example of one working in resource and one not working as textarea but does work as textinput in a custom page.
https://github.com/griggsk/osp/blob/main/app/Filament/Resources/ChildResource.php
https://github.com/griggsk/osp/blob/main/app/Livewire/Questionnaire/StartQuestionnaire.php

And here is the example I reference above: https://github.com/griggsk/osp/blob/main/app/Livewire/Questionnaire/CompleteQuestionnaire.php

**Comment by arne-s** [comment]
I have the same issue, has this been fixed in the latest version? Is there a workaround?

**Comment by jeromevillaver** [comment]
We faced same issue.. what we did is just re-install the filament so that it will upgrade and also the packages in it.
